config/_shared.json

```diff
@@ -1,3 +1,4 @@
 {
-	"discover_blog_id": 53424024
+	"discover_blog_id": 53424024,
+	"project": "project"
```

Comment:
We need to explicitly set `wpcom.js` otherwise, it tries to load our REST API library ;)